src/account.js

```diff
@@ -23,6 +24,8 @@ class Account {
     this._clientPaychan = null
     this._funding = null
     this._claimIntervalId = null
+
+    this._log = createLogger(DEBUG_NAMESPACE)
```

Comment:
Why is log a parameter in the constructor if w're creating our own logger? It might be better to use the logger from the plugin. It would be ideal if we could add an `account` segment to the plugin logger's namespace, but that might not be possible with the current structure of the logger

Reply:
I did this to keep the `account` namespace, but it might indeed be better to just pass in the same logger as the plugin. I'm not sure how to maintain the original namespace without creating another logger, especially when in the case of riverpig it's already fully instantiated.

Reply:
yeah, I'd say just pass in the logger from the plugin and use that